packages/css-data/src/__generated__/property-value-descriptions.ts

```diff
@@ -0,0 +1,4574 @@
+// This file was auto-generated
+export const properties = {
```

Comment:
we are missign properties like "background", "border" etc because we are not using shortcuts in the data, but we have those as categories and need tooltips for them

Reply:
I thought you didn't support shorthand

Reply:
we don't, but we show the tooltip for example on background category etc